data/gamerules.txt

```diff
@@ -24,9 +24,9 @@ gamerules
 	"universal ramscoop" false
 	
 	# An attempt to spawn a person ship happens on average every this many frames.
-	# DEFAULT: 72000 frames (twenty minutes)
+	# DEFAULT: 9000 frames (two and a half minutes)
```

Comment:
```suggestion
	# DEFAULT: 36000 frames (ten minutes)
```

I like the sentiment, but I think boosting the person spawn rate shorter than ten minutes is premature at this point. Probably another twenty people or so before we can start looking at making it shorter than ten.